pkg/scheduler/core/extender.go

```diff
@@ -29,8 +29,9 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	restclient "k8s.io/client-go/rest"
 	extenderv1 "k8s.io/kube-scheduler/extender/v1"
+
```

Comment:
we usually only leave an empty line  between builtin imports and the k8s one, so this line is not needed.

Reply:
all other files as well